### DISCUSSION TITLE
Generate sourcemaps for easier debugging

### DISCUSSION THREAD

**Discussion by lannka** [CoderLine/alphaTab#582] [Ideas]
Shall we enable sourcemaps for easier debugging?
I have a simple PR (#583) for that if there is no objections.

**Comment by Danielku15** [comment]
I have disabled source maps on purpose: In my development setup I use the browser development tools to debug and there I have terrible experience with source maps. They rather break proper debugging possibilities than they bring any value. Usually breakpoints do not work anymore and DevTools start to hang when you use source maps. 

With the new ES6 compilation since 1.2 the final alphaTab.js contains clean and readable code. Mapping back any error location to the TypeScript codebase is usually a matter of seconds when searching for classes/methods in VsCode.
